src/main/java/org/cryptomator/ui/mainwindow/VaultListController.java

```diff
@@ -109,6 +113,9 @@ public void initialize() {
 		vaultList.setItems(vaults);
 		vaultList.setCellFactory(cellFactory);
 
+		vaultList.fixedCellSizeProperty().bind(Bindings.createDoubleBinding(() ->
+				settings.compactMode.get() ? 30.0 : 60.0, settings.compactMode));
+
```

Comment:
Do not bind it here. Create an explicit Binding (e.g. `settings.compactMode.map(...)`) and store it as an instance variable. Then create a getter and bind the fixed cell size property in the fxml.